ldk/javascript/src/types/oliveHelps/index.d.ts

```diff
@@ -424,7 +433,13 @@ declare namespace OliveHelps {
     | Telephone
     | TextInput;
 
-  type Components = Box | ChildComponents | CollapseBox;
+  type Form = Component<WhisperComponentType.Form> & {
+    children: Array<ChildComponents>;
+    onSubmit: (values: Map<string, any>) => void;
+    // TODO: Make submit button configurable
```

Comment:
Do you still need to do this todo?

Reply:
I'll remove it and create a card for it.